packages/lib/src/vis/heatmap/HeatmapMesh.tsx

```diff
@@ -201,26 +141,27 @@ function HeatmapMesh(props: Props) {
     fragmentShader: `
       uniform sampler2D data;
       uniform sampler2D colorMap;
-      uniform int scaleType;
-      uniform float alphaMin;
+
+      uniform float min;
+      uniform float oneOverRange;
+      uniform float gammaExponent;
+
       uniform sampler2D alpha;
+      uniform float alphaMin;
       uniform float oneOverAlphaRange;
       uniform int withAlpha;
 
       const vec4 nanColor = vec4(255, 255, 255, 1);
+      const float oneOverLog10 = 0.43429448190325176;
```

Comment:
Required for log/symlog. Again, I think the performance impact of declaring this regardless of the scale is negligible.